modules/kv-state-cache/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+file(GLOB VINEYARD_KV_STATE_CACHE_SRCS "${CMAKE_CURRENT_SOURCE_DIR}"
+                                "ds/*.cc"
+                                "ds/*.h"
+                                "radix-tree/*.cc"
+                                "radix-tree/*.h"
+                                "utils/*.cc"
+                                "utils/*.h"
+                                "strategy/*.cc"
+                                "strategy/*.h"
+)
+
+add_library(vineyard_kv_state_cache ${VINEYARD_KV_STATE_CACHE_SRCS})
+target_link_libraries(vineyard_kv_state_cache PUBLIC vineyard_client vineyard_basic)
```

Comment:
Rename the target to `vineyard_llm_cache`.

Reply:
Done